commands/volumes/volume-create.go

```diff
@@ -120,6 +121,39 @@ func rollBackVolumeCreate(c *context.Context) error {
 	return nil
 }
 
+func registerVolCreateStepFuncs() {
+	var sfs = []struct {
+		name string
+		sf   transaction.StepFunc
+	}{
+		{"vol-create.Stage", validateVolumeCreate},
+		{"vol-create.Commit", generateVolfiles},
+		{"vol-create.Store", storeVolume},
+		{"vol-create.Rollback", rollBackVolumeCreate},
```

Comment:
On a broader note, shouldn't volfiles be written into etcd store instead of storing local filesystem ? That way any glusterd instance can serve volfile of any volume.